modules/network-observability-without-loki.adoc

```diff
@@ -0,0 +1,19 @@
+// module included in the following assemblies:
+// networking/network_observability/installing-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-without-loki_{context}"]
+= Network Observability without Loki
+You can use Network Observability without Loki by omitting the Loki installation steps and only using exporters, such as Kafka or IPFIX. The following table compares available features with and without Loki:
+
+.Comparison of feature availability with and without Loki
```

Comment:
Nice!

Reply:
`X` might be self-explanatory, but do we need to state:

> , where `X` indicates Network Oberservebility supports a Loki or non-Loki capability.

Reply:
s/by omitting the Loki installation steps/not performing the Loki installation steps in the following section

Reply:
I find `x` self-explanatory, especially in context with N/A. However, a check mark might be better

Reply:
I'm just going to leave the X. I think its self-explanatory and I also don't feel like making an icon for this

Reply:
I revisit this suggestion and added checks and x's because I have come to think this is a worthwhile and clearer presentation. Thanks for putting the idea in my brain :)